spark/Debugging-extended-Spark-History-Server.md

```diff
@@ -0,0 +1,202 @@
+Extended Spark History Server is built on top of the community version of Spark History Server and provide advanced features. The current extension includes data tab and graph tab. Data tab shows the input/output data source of the Spark job including table operations. This allows users to validate the output, preview input schema/data to facilitate the development. Graph tab provides job graph, playback and heatmap. This helps users to understand the bottleneck, monitor and diagonose issues. Please check [Extended Spark UI documentation](https://go.microsoft.com/fwlink/?linkid=2005401) for detail information.
+
+## 1. Hotfix extended Spark History Server
+
+To hotfix with new version, use the script below which will download and update spark-enhancement.jar*.
+
+**upgrade_spark_enhancement.sh**:
```

Comment:
No need to provide the entire script here. You can add the link to script action repo.

Reply:
Sure, please review this PR, once commited, I will update the script.

Reply:
hdinsight/script-actions#13